spyder_kernels/console/kernel.py

```diff
@@ -27,7 +29,7 @@
 from zmq.utils.garbage import gc
 
 # Local imports
-from spyder_kernels.comms.frontendcomm import FrontendComm
+from spyder_kernels.comms.frontendcomm import FrontendComm, CommError
```

Comment:
```suggestion
from spyder_kernels.comms.frontendcomm import FrontendComm
```

Seems to not be used anymore.